install-script.md

```diff
@@ -34,269 +34,14 @@ Be aware of the following:
 
 > SSH in using Putty, mRemoteNG, or any other SSH tool. Note that most tools support saving your connection.{.is-info}
 
-- Once SSHed in type the below to create the installation script in your current directory
+- Once SSHed in type the below to download the installation script in your current directory
 
 ```bash
-nano ArrInstall.sh
+curl -o starrInstall.sh https://raw.githubusercontent.com/Servarr/Wiki/master/servarr/servarr-install-script.sh
```

Comment:
```suggestion
curl -o servarr-install-script.sh https://raw.githubusercontent.com/Servarr/Wiki/master/servarr/servarr-install-script.sh
```

Why change the name on download? starr doesn't mean so much when its referred to as Servarr Installation Script